packages/gatsby-source-lever/normalize.js

```diff
@@ -0,0 +1,214 @@
+"use strict";
```

Comment:
.gitignore fail

Reply:
i'mm go through and try and address all these in a PR